libs/openFrameworks/math/ofQuaternion.h

```diff
@@ -199,21 +199,21 @@ ofQuaternion::ofQuaternion() {
 
 
 //----------------------------------------
-ostream& ofQuaternion::operator<<(ostream& os) {
-    os << _v.x << ", " << _v.y << ", " << _v.z << ", " << _v.w;
+ostream& operator<<(ostream& os, const ofQuaternion &q) {
```

Comment:
yeah!
i think so too. and i post new PR #4729 with those fix.
i pray to god of ci-travis...